packages/graphql-mesh-server/assets/nginx/nginx.conf

```diff
@@ -0,0 +1,53 @@
+events {
+  worker_connections 1024;
+}
+
+http {
+  log_format json_combined escape=json
```

Comment:
JSON logging threw me, but this seems correct.

Reply:
Yeah it looks a bit funky in the config but it seems to work. We want to use JSON so we have better CloudWatch query support: https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_AnalyzeLogData-discoverable-fields.html